packages/DApp/src/components/Rules.tsx

```diff
@@ -17,45 +17,52 @@ export function Rules() {
         <RuleIcon>👍</RuleIcon>
         <RuleText>
           If a vote to add ends in favour of a community, the community will be added to the directory. Otherwise, a new
-          vote for the addition of that community can be started after 30 days.
+          vote for the addition of that community can be started after 2 days.
         </RuleText>
       </Rule>
       <Rule>
         <RuleIcon>🗑️</RuleIcon>
         <RuleText>
           If a vote to remove a community ends in favour of the removal, the community will disappear from the
-          directory. Otherwise, a new vote for removal can be submitted after about 30 days.
+          directory. Otherwise, a new vote for removal can be submitted after about 2 days.
         </RuleText>
       </Rule>
-      <Rule>
+      {/* <Rule>
         <RuleIcon>✌️️</RuleIcon>
         <RuleText>
           The minimum amount of SNT needed to start a new vote doubles with every failed vote attempt (for both addition
           and removal votes).
         </RuleText>
+      </Rule> */}
+      <Rule>
+        <RuleIcon>⏳</RuleIcon>
+        <RuleText>
+          Each vote lasts for two days. When the voting period ends, the verification period starts, someone
+          has to cast the votes.
+        </RuleText>
       </Rule>
       <Rule>
         <RuleIcon>⏳</RuleIcon>
         <RuleText>
-          Each vote lasts for two weeks. When the voting period ends, someone has to finalize the vote with a
+          Each verification period lasts for one day. When the verification ends, someone has to finalize the vote with a
           finalization transaction.
         </RuleText>
       </Rule>
-      <Rule>
+      {/* <Rule>
         <RuleIcon>⚠️</RuleIcon>
         <RuleText>
           If a single vote of more than 2,000,000 SNT votes for is made in favor of the removal of a community, the
           remaining vote duration shortens to 24 hours. This shortening of the vote duration can be reversed if a single
           vote of more than 2,000,000 SNT is made against the removal.
         </RuleText>
-      </Rule>
+      </Rule> */}
       <Rule>
         <RuleIcon>⭐</RuleIcon>
         <RuleText>
-          There is an always ongoing vote for which communities from the directory should appear in the Weekly Featured
-          section. Every week, five communities with the most votes are added to the section, replacing the five
-          communities that were featured the previous week. After the community leaves Weekly Featured, it cannot get
-          there again for three weeks.
+          There is an always ongoing vote for which communities from the directory should appear in the Featured
+          section. Every three days, three communities with the most votes are added to the section, replacing the three
+          communities that were featured previously. After the community leaves Featured section, it cannot get
+          there again for around a week.
```

Comment:
"around a week"? we cannot be specific?

Reply:
Not really, we don't have control over when the vote is started and when it is finalized, as it is the manual action required by peers.

Reply:
> After the community leaves Featured section, it cannot get
          there again for around a week.

Where is this evaluated in the code?

Reply:
It is the contract that validates it: https://github.com/status-im/community-dapp/blob/30e8b267d67379d0eb2bc6da598615f5586baf63/packages/contracts/contracts/FeaturedVotingContract.sol#L185-L196